### PR TITLE
Loyalty readme updates 

### PR DESCRIPTION
**Pull request by ShannonCanTech**
New loyalty survey links added to each MSAL library's README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,10 +58,17 @@ Please check the [roadmap](https://github.com/AzureAD/microsoft-authentication-l
...
+## Share Your Experience
+How easy was it to use this library? And valuable did you find it? Annonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
+
 ## Contribute
 
 We enthusiastically welcome contributions and feedback. Please read the [contributing guide](contributing.md) before you begin.
 
+## Share your experience
+
+How easy was it to use this library? And valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
```
Comment (jasonnutter) [inline README.md:70]: Looks like this section was added twice to this file.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[Got feedback?](https://ncv.microsoft.com/NhQGJ4UuqI)
```
Comment (jasonnutter) [inline README.md:1]: Is this needed here, given the links below?

File: lib/msal-angularjs/README.md [hunk 3]
```diff
@@ -335,6 +337,9 @@ We highly recommend you ask your questions on Stack Overflow first and browse ex
 - [GitHub Issues](../../issues) for reporting a bug or feature requests
 
 - [User Voice page](https://feedback.azure.com/forums/169401-azure-active-directory) to provide recommendations and/or feedback
+    
+## Share Your Experience
+How easy was it to use this library? And valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/98iH6JcTda).
```
Comment (jasonnutter) [inline lib/msal-angularjs/README.md:342]: Same questions re: link at top and this section being included twice.

**Comment by jasonnutter** [comment]
Looks like there are merge conflicts, please update from dev!

**Review by jasonnutter — CHANGES_REQUESTED** [review]
A few questions, plus merge conflicts. Thanks!

**Comment by OwenWheatley** [comment]
#3866 fixes issues raised

File: README.md [hunk 4]
```diff
@@ -56,10 +58,17 @@ Please check the [roadmap](https://github.com/AzureAD/microsoft-authentication-l
 
 - [User Voice page](https://feedback.azure.com/forums/169401-azure-active-directory) to provide recommendations and/or feedback.
 
+## Share Your Experience
+How easy was it to use this library? And valuable did you find it? Annonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
```
Comment (sameerag) [inline README.md:62]:
```suggestion
How easy was it to use this library? And how valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
```

File: lib/msal-angularjs/README.md (hunk 3 above)
Comment (sameerag) [reply lib/msal-angularjs/README.md:342]: Same edit:
```suggestion
How easy was it to use this library? How valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/98iH6JcTda).
```

File: lib/msal-angularjs/README.md [hunk 5]
```diff
@@ -358,6 +363,10 @@ This library controls how users sign-in and access services. We recommend you al
 ## Security Reporting
 
 If you find a security issue with our libraries or services please report it to [secure@microsoft.com](mailto:secure@microsoft.com) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://technet.microsoft.com/security/dd252948) and subscribing to Security Advisory Alerts.
+    
+## Share your experience
+    
+How easy was it to use this library? And valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/98iH6JcTda).
```
Comment (sameerag) [inline lib/msal-angularjs/README.md:369]:
```suggestion
How easy was it to use this library? How valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/98iH6JcTda).
```

File: README.md (hunk 1 above)
Comment (ShannonCanTech) [reply README.md:70]: Thanks Jason, updating in new revision.

File: lib/msal-react/README.md [hunk 6]
```diff
@@ -4,8 +4,13 @@
...
 [![codecov](https://codecov.io/gh/AzureAD/microsoft-authentication-library-for-js/branch/dev/graph/badge.svg?flag=msal-react)](https://codecov.io/gh/AzureAD/microsoft-authentication-library-for-js)
 
+<<<<<<< HEAD
+| <a href="https://aka.ms/aaddevv2" target="_blank">AAD Docs</a> | <a href="https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_react.html" target="_blank">Library Reference</a> | <a href="https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples" target="blank">Samples</a> | <a href="https://ncv.microsoft.com/F22aBJtI8l" target="_blank">Feedback</a>|
+| --- | --- | --- | --- |
+=======
 | <a href="https://docs.microsoft.com/azure/active-directory/develop/tutorial-v2-react" target="blank">Getting Started</a> | <a href="https://aka.ms/aaddevv2" target="_blank">AAD Docs</a> | <a href="https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_react.html" target="_blank">Library Reference</a> | <a href="https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples" target="blank">Samples</a>
 |--- | --- | --- | --- |
+>>>>>>> origin/dev
```
Comment (tnorling) [inline lib/msal-react/README.md:13]: Resolve this merge conflict
```suggestion
| <a href="https://docs.microsoft.com/azure/active-directory/develop/tutorial-v2-react" target="blank">Getting Started</a> | <a href="https://aka.ms/aaddevv2" target="_blank">AAD Docs</a> | <a href="https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_react.html" target="_blank">Library Reference</a> | <a href="https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples" target="blank">Samples</a>| <a href="https://ncv.microsoft.com/F22aBJtI8l" target="_blank">Feedback</a>
| --- | --- | --- | --- | --- |
```

File: lib/msal-angular/README.md [hunk 7]
```diff
@@ -4,8 +4,13 @@
...
+| --- | --- | --- | --- | --- | --- |
+=======
 | <a href="https://docs.microsoft.com/azure/active-directory/develop/tutorial-v2-angular-auth-code" target="blank">Getting Started</a>| <a href="https://aka.ms/aaddevv2" target="_blank">AAD Docs</a> | <a href="https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_angular.html" target="_blank">Library Reference</a> | [Support](README.md#community-help-and-support) | <a href="https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v2-samples" target="blank">Samples</a>
 | --- | --- | --- | --- | --- |
+>>>>>>> origin/dev
```
Comment (tnorling) [inline lib/msal-angular/README.md:13]: Resolve this merge conflict
```suggestion
| <a href="https://docs.microsoft.com/azure/active-directory/develop/tutorial-v2-angular-auth-code" target="blank">Getting Started</a>| <a href="https://aka.ms/aaddevv2" target="_blank">AAD Docs</a> | <a href="https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_angular.html" target="_blank">Library Reference</a> | [Support](README.md#community-help-and-support) | <a href="https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v2-samples" target="blank">Samples</a> | <a href="https://ncv.microsoft.com/0v3roKKyf8" target="_blank">Feedback</a>
| --- | --- | --- | --- | --- | --- |
```

File: README.md [hunk 8]
```diff
@@ -68,6 +68,10 @@ Please check the [roadmap](https://github.com/AzureAD/microsoft-authentication-l
 
 We enthusiastically welcome contributions and feedback. Please read the [contributing guide](contributing.md) before you begin.
 
+## Share your experience
+
+How easy was it to use this library? And valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
```
Comment (tnorling) [inline README.md:73]:
```suggestion
How easy was it to use this library? How valuable did you find it? Anonymously share your experience with us [here](https://ncv.microsoft.com/NhQGJ4UuqI).
```

**Comment by jasonnutter** [comment]
@ShannonCanTech Still planning to complete this PR? If not, please close, thanks!

**Comment by jasonnutter** [comment]
Closing due to inactivity. Please open a new PR, thanks!

**Pull request closed without merging**